scripts/package/add_packages.ps1

```diff
@@ -0,0 +1,20 @@
+param (
+    [Parameter(Mandatory=$true, Position=0)] [string] $DOTNET_PROJECT,
+    [Parameter(Mandatory=$true, Position=1)] [string] $PACKAGE_SRC
+)
+
```

Comment:
nit: can we add the same error checking we have in the bash script here too?

Reply:
Setting these params as mandatory will automatically throw an error message saying the field is missing, so it may be redundant to do so. Example: `1: Cannot bind argument to parameter 'DOTNET_SOLUTION' because it is an empty string.` Unless we want those error statements in particular, then yes, I can add them!

Reply:
Update: ended up making this change so the script execution is as close to the other ones as possible 😄